src/filefinder.cpp

```diff
@@ -61,6 +61,13 @@ namespace {
 	std::shared_ptr<Filesystem> root_fs;
 	FilesystemView game_fs;
 	FilesystemView save_fs;
+
+	constexpr const auto IMG_TYPES = Utils::MakeSvArray(".bmp",  ".png", ".xyz");
+	constexpr const auto MUSIC_TYPES = Utils::MakeSvArray(
+			".opus", ".oga", ".ogg", ".wav", ".mid", ".midi", ".mp3", ".wma");
+	constexpr const auto SOUND_TYPES = Utils::MakeSvArray(
+			".opus", ".oga", ".ogg", ".wav", ".mp3", ".wma");
+	constexpr const auto FONTS_TYPES = Utils::MakeSvArray(".ttf", ".ttc", ".otf", ".fon", ".fnt", ".bdf");
```

Comment:
.woff and .woff2 OpenType formats are useful for significantly smaller emscripten asset downloads (may require extra freetype dependencies, e.g. brotli).

Reply:
The FileFinder code aborts early when the directory is missing so for most games adding more extensions won't hurt. I add a commit :)